pyquil/api/_compiler_client.py

```diff
@@ -18,7 +18,7 @@
 from dataclasses import dataclass
 from typing import Iterator, List, Optional
 
-import qcs_sdk
+from qcs_sdk.compiler.quilc import get_version_info
```

Comment:
Had to patch this here since the import paths changed in [qcs-sdk-rust#255](https://github.com/rigetti/qcs-sdk-rust/pull/255).